week3/homework/src/index.js

```diff
@@ -1,3 +1,144 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express')
+const app = express()
+const port = 3000;
+const fs = require('fs');
+const uuid = require('uuid/v4');
+const bodyParser = require("body-parser");
+
+
+app.use(bodyParser.json());
+
+
+
+app.get('/todos', (request, response) => {
+  fs.readFileSync('./todo.txt', 'utf8', function (error, data = []) {
```

Comment:
Make sure to use `todo.json` instead of txt.
Parse the JSON to a JavaScript object and use that in your code. That way you do not have to use the new lines `\n` you are using now.